presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -571,13 +589,15 @@ Procedures
     * ``DROP``: drop any partitions that exist in the metastore but not on the file system.
     * ``FULL``: perform both ``ADD`` and ``DROP``.
 
-Examples
---------
+Working with external data
+--------------------------
 
 The Hive connector supports querying and manipulating Hive tables and schemas
 (databases). While some uncommon operations will need to be performed using
 Hive directly, most operations can be performed using Presto.
 
+The examples shown here should work on Google Cloud Storage after replacing ``s3://`` with ``gs://``.
```

Comment:
Let's move this sentence to the end, after all the examples, so as not to distract from the introductory paragraph. I find it confusing here since only two of the examples mention S3.